.env.docker.example

```
@@ -0,0 +1,68 @@
+# --- Runtime ------------------------------------------------------------------
+ENVIRONMENT=production
+LOG_LEVEL=INFO
+APP_PORT=8000
+HTTP_PORT=80
+# Dependency set baked into the image: all | api,mcp | worker | ml ...
+INSTALL_TARGET=all
+
+# --- Infrastructure secrets (auto-filled by `make docker-secrets`) ------------
+POSTGRES_PASSWORD=__CHANGE_ME__
+REDIS_PASSWORD=__CHANGE_ME__
+GRAFANA_ADMIN_PASSWORD=__CHANGE_ME__
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_ADMIN_PASSWORD key should go before the POSTGRES_PASSWORD key

(UnorderedKey)